src/opnsense/mvc/app/models/OPNsense/Monit/Monit.xml

```diff
@@ -47,6 +47,23 @@
             <default>0</default>
             <Required>Y</Required>
          </ssl>
+         <sslversion type="OptionField">
+            <default>auto</default>
+            <Required>Y</Required>
+            <OptionValues>
+               <auto>AUTO</auto>
+               <sslv2>SSLV2</sslv2>
+               <sslv3>SSLV3</sslv3>
+               <tlsv1>TLSV1</tlsv1>
+               <tlsv11>TLSV11</tlsv11>
+               <tlsv12>TLSV12</tlsv12>
+               <tlsv13>TLSV13</tlsv13>
+            </OptionValues>
+         </sslversion>
+         <sslverify type="BooleanField">
+            <Required>N</Required>
```

Comment:
since it's easy to turn off after update it should be Required Y and default 1 for proper migration and security standard